### PR TITLE
Add Translation to filter values & Renaming

### PR DESCRIPTION
**Pull request by MicheleK17**
Ticket: #255
Change-Id: I8de64dc2e67246cae025b77bac2c7395c126e7e8

### CODE REVIEW

File: src/locales/en/QueryPage.json [hunk 1]
```diff
@@ -49,6 +49,179 @@
...
+            "Muskel": "Muscle",
+            "Petersilie": "Parsley",
+            "Alleinfuttermittel": "Complete feedingstuffs",
+            "Babyspinat": "Baby spinach"
+        },
+        "samplingContext": {
+            "Zoonosen-Monitoring": "Zoonoses-Monitoring",
+            "Salm-Bekämpfung": "Salm-control programs"
+        },
+        "federalState": {
```
Comment (GerberI) [inline src/locales/en/QueryPage.json:96]: didn't know how funny those translation are :)

**Review by GerberI — CHANGES_REQUESTED** [review]

File: src/locales/de/QueryPage.json [hunk 2]
```diff
@@ -38,17 +38,190 @@
...
+            "Tatar / Schabefleisch": "Tatar / Schabefleisch",
+            "Nasentupfer": "Nasentupfer",
+            "Muskel": "Muskel",
+            "Petersilie": "Petersilie",
+            "Alleinfuttermittel": "Alleinfuttermittel",
+            "Babyspinat": "Babyspinat"
+        },
+        "samplingContext": {
+            "Zoonosen-Monitoring": "Zoonosen-Monitoring",
+            "Salm-Bekämpfung": "Salm-Bekämpfung"
```
[the remark this answers is not in the document]
Comment (MicheleK17) [reply src/locales/de/QueryPage.json:94]: Nein, Salm-Bekämpfung ist richtig :-)

File: src/locales/de/QueryPage.json [hunk 3]
```diff
@@ -38,17 +38,190 @@
...
+            "Nordrhein-Westfalen": "Nordrhein-Westfalen",
+            "Niedersachsen": "Niedersachsen",
+            "Saarland": "Saarland",
+            "Hamburg": "Hamburg",
+            "Schleswig-Holstein": "Schleswig-Holstein",
+            "Sachsen": "Sachsen",
+            "Mecklenburg-Vorpommern": "Mecklenburg-Vorpommern",
+            "Rheinland-Pfalz": "Rheinland-Pfalz",
+            "Sachsen-Anhalt": "Sachsen-Anhalt",
+            "Thueringen": "Thueringen"
```
[the remark this answers is not in the document]
Comment (MicheleK17) [reply src/locales/de/QueryPage.json]: Done.

File: src/locales/de/QueryPage.json [hunk 4]
```diff
@@ -38,17 +38,190 @@
...
+            "Petersilie": "Petersilie",
+            "Alleinfuttermittel": "Alleinfuttermittel",
+            "Babyspinat": "Babyspinat"
+        },
+        "samplingContext": {
+            "Zoonosen-Monitoring": "Zoonosen-Monitoring",
+            "Salm-Bekämpfung": "Salm-Bekämpfung"
+        },
+        "federalState": {
+            "Baden-Wuerttemberg": "Baden-Wuerttemberg",
```
[the remark this answers is not in the document]
Comment (MicheleK17) [reply src/locales/de/QueryPage.json]: Done.

**Review by GerberI — APPROVED** [review]
Commit: Add Ü

**Pull request merged**